tests/integration/test_api/test_config/test_max_upload_size/test_max_upload_size.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from os.path import join, dirname, realpath
+from uuid import uuid4
+
+import pytest
+import requests
+
+from wazuh_testing.tools.configuration import check_apply_test, get_api_conf
+
+# Marks
+
+pytestmark = pytest.mark.server
+
+# Configurations
+
+test_data_path = join(dirname(realpath(__file__)), 'data')
+configurations_path = join(test_data_path, 'conf.yaml')
+configuration = get_api_conf(configurations_path)
+
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configuration)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Tests
+
+@pytest.mark.parametrize('tags_to_apply, expected_status_code', [
+    ({'limitless_upload_size'}, 200),
+    ({'low_upload_size'}, 413)
+])
+def test_max_upload_size(tags_to_apply, expected_status_code, get_configuration, configure_api_environment, restart_api,
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over.  That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
In case of the need of all Wazuh daemon, the test should disable every module not necessary for the testing in `ossec.conf`.

The reasons for this change are:
- Faster testing
- More encapsulated tests